src/lib/logical_query_plan/drop_view_node.hpp

```diff
@@ -12,18 +12,28 @@ namespace opossum {
  */
 class DropViewNode : public AbstractLQPNode {
  public:
+  static std::shared_ptr<DropViewNode> make(const std::string& view_name);
+
   explicit DropViewNode(const std::string& view_name);
 
   std::string description() const override;
   bool subtree_is_read_only() const override;
+  const std::vector<std::string>& output_column_names() const override;
+
+  bool shallow_equals(const AbstractLQPNode& rhs) const override;
 
   const std::string& view_name() const;
 
  protected:
   std::shared_ptr<AbstractLQPNode> _deep_copy_impl(
       const std::shared_ptr<AbstractLQPNode>& copied_left_child,
       const std::shared_ptr<AbstractLQPNode>& copied_right_child) const override;
+
+ private:
   const std::string _view_name;
+
+  // Need an instance since we're returning a reference in the getter
+  std::vector<std::string> _output_column_names_dummy;
```

Comment:
`static`?

Reply:
It's just space on the stack, not worth the text for "static" and the definition in the cpp.

Reply:
okay new idea. create a static variable within the method and return it.

Reply:
betta